scripts/linters/pylint_extensions.py

```diff
@@ -829,6 +834,11 @@ def check_docstring_structure(self, node):
         """
         if node.doc:
             docstring = node.doc.splitlines()
+            for line in docstring:
+                if (line.strip().startswith(
+                        (b'Args:', b'Raises:', b'Returns:', b'Yields:')) and
+                        len(line.split()) != 1):
```

Comment:
Something like `Returns:str` won't be caught by this, right (since you split by spaces)? Do you see a check happening before this which can ensure this is caught? Otherwise I would prefer having check in which you do an exact match with these names i.e `line.strip() == 'Args'`, so that we will be sure about what this line contains, and successfully eliminate rare but wrong examples like `Returns:str`.